apps/fulfillment/src/index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
     <oryx-app></oryx-app>
-    <script type="module" src="/src/app.ts"></script>
+    <script type="module" src="/app.ts"></script>
```

Comment:
```suggestion
    <script type="module" src="./app.ts"></script>
```

nitpick: just so we can cmd-click in the IDE